soc/arm/nuvoton_npcx/npcx7/Kconfig.defconfig.series

```diff
@@ -50,6 +50,14 @@ config TACH_NPCX
 	default y
 	depends on SENSOR
 
+config KSCAN_NPCX
+	default y
+	depends on KSCAN
```

Comment:
Delete - put the depends on the `KSCAN_MATRIX_NPCX` defintion.

Reply:
Done. Also delete this in soc/arm/nuvoton_npcx/npcx9/Kconfig.defconfig.series